exporter/collector/logs_test.go

```diff
@@ -396,6 +395,103 @@ func TestLogMapping(t *testing.T) {
 			},
 			maxEntrySize: defaultMaxEntrySize,
 		},
+		{
+			name: "log with bad source location (bytes)",
```

Comment:
What makes this test "bad"?

Reply:
It's because the value provided for `gcp.source_location` can't actually be parsed, the json Unmarshal fails due to an invalid type in one of the fields. Is there a nicer way I should say that concisely in the test or should I add a comment?

Reply:
`"log with wrong type in sourceLocation (bytes)"`, perhaps?